src/component/menu/ApiDetails/ApiDetails.tsx

```diff
@@ -0,0 +1,106 @@
+import useUiConfig from '../../../hooks/api/getters/useUiConfig/useUiConfig';
+import { ReactElement } from 'react';
+import ConditionallyRender from '../../common/ConditionallyRender';
+import { IVersionInfo } from '../../../interfaces/uiConfig';
+
+interface PartialUiConfig {
+	name: string;
+	version: string;
+	slogan?: string;
+	environment?: string;
+	versionInfo?: IVersionInfo;
+}
+
+interface Props {
+	uiConfig: PartialUiConfig;
+}
+
+export function ApiDetailsContainer(): ReactElement {
```

Comment:
Nice. This makes this component super testable. But perhaps we could just use the hook directly in the Footer and pass in the data from there?